### ISSUE TITLE
alerts: suppress alerts when PlatformCluster_FederationScrapeJobFailing is firing

### ISSUE THREAD

**Issue by stephen-soltesz** [m-lab/prometheus-support#683]
The alerts below all depend on the platform cluster being up.

If `PlatformCluster_FederationScrapeJobFailing` is firing, then the following alerts should be suppressed.

```
MachineRunningWithoutK8sNode (390 active)
PlatformCluster_CadvisorMissing (1 active)
PlatformCluster_CoreServices_CollectdMlabMissing (1 active)
PlatformCluster_FluentdMissing (1 active)
PlatformCluster_HostExperimentMissing (1 active)
PlatformCluster_NdtMissing (1 active)
PlatformCluster_NeubotMissing (1 active)
PlatformCluster_NodeExporterMissing (1 active)
PlatformCluster_WeheMissing (1 active)
```

**Comment by nkinkade** [comment]
@stephen-soltesz: Does this issue mean that the existing inhibit rule is apparently not working?

https://github.com/m-lab/prometheus-support/blob/master/config/federation/alertmanager/config.yml.template#L83

**Comment by stephen-soltesz** [comment]
Unfortunately, I don't remember where I got that list. I looked in slack for the alert history around May 18th, and didn't see the PlatformCluster_* alerts there, but there were a bunch of alerts that day. It's possible I pulled them from Prometheus directly without thinking about the fact that those don't matter as much as what AM does with them.

Maybe just `[FIRING:390] MachineRunningWithoutK8sNode`?
